src/suggestions.ts

```diff
@@ -110,11 +111,47 @@ export async function postPatchSuggestion(
       ...lineArgs,
       commit_id: pull_request.head.sha,
       pull_number: pull_request.number,
-      body: `To fix violation '${ps.violationSpec}', Sorald suggests the following:
-${ps.suggestion}
-`,
+      body: await generateSuggestionMessage(ps),
       path: ps.file.toString(),
       start_side: 'RIGHT'
     });
   }
 }
+
+/**
+ * Generate a full, human-friendly suggestion message based on a patch suggestion.
+ *
+ * @param ps - A patch suggestion
+ * @returns Fulfills to a suggestion message
+ */
+export async function generateSuggestionMessage(
+  ps: PatchSuggestion
+): Promise<string> {
+  const [ruleKey] = ps.violationSpec.split(':');
+  const sonarVersion = '6.9.0.23563';
```

Comment:
Is this something you will have to keep in sync with sorald's development?

Reply:
That'd be a "maybe". The rule descriptions very rarely change, and new rules are fairly rarely added. The ideal solution would be to fetch the current Sonar version from Sorald, but that's not currently possible.

Reply:
> The ideal solution would be to fetch the current Sonar version from Sorald, but that's not currently possible.

Yeah, I was about to suggest that but if it is not changing frequently, you can just pin the version here.

Reply:
Yeah I think that's the path of least resistance. It's not ideal, but putting in the work to resolving it dynamically is probably never going to pay off.